### PR TITLE
Add check items in register-EN

### PR DESCRIPTION
**Pull request by memorycancel**
As #29 merged.

Add check items in register-EN, @lesterli please review, thanks.

### CODE REVIEW

File: src/guides/appchain-register.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
 ## Appchain Register
 
+### Appchain check items
```
Comment (HinadaC) [inline src/guides/appchain-register.md:3]: checklist

Comment (lesterli) [inline src/guides/appchain-register.md]: Checklist before registration

File: src/guides/appchain-register.md [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+
+This section aims to list the items that the appchain team needs to check before appchain registration to ensure a smooth launch.
+
++ The appchain is using the latest version of the barnacle/barnacle-evm template
++ The version of Substrate used in Cargo.toml of appchain should be consistent with the template.
++ The runtime code should be consistent with the definition of the integrated Pallets in the template construct_runtime!
```
Comment (lesterli) [inline src/guides/appchain-register.md]: The below pallets defined in the `construct_runtime!` part in the Appchain runtime code (runtime/src/lib.rs) should be consistent with the template.

File: src/guides/appchain-register.md [hunk 3]
```diff
@@ -1,5 +1,59 @@
...
+		MmrLeaf: pallet_beefy_mmr,
+		Uniques: pallet_uniques,
+		Sudo: pallet_sudo,
+		// Include the custom logic from the pallet-template in the runtime.
+		TemplateModule: pallet_template,
+	}
+);
+```
+
++ The block generation time of the appchain is consistent with the template, set to 6s.
```
Comment (lesterli) [inline src/guides/appchain-register.md]: block time

File: src/guides/appchain-register.md [hunk 4]
```diff
@@ -1,5 +1,59 @@
...
+
+```rust
+pub const MILLISECS_PER_BLOCK: Moment = 6000;
+pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 4 * HOURS;
+```
++ In appchain Spec JSON file, ss58Format is 42, if you do not need to transfer the stablecoin assets (USDN) of the main chain (NEAR network) to appchain, please configure assetIdByTokenId to be empty.
```
Comment (lesterli) [inline src/guides/appchain-register.md]: Appchain chain spec

Comment (lesterli) [reply src/guides/appchain-register.md]: set `assetIdByTokenId` with []

File: src/guides/appchain-register.md [hunk 5]
```diff
@@ -1,5 +1,59 @@
...
+
+```
+"assetIdByTokenId": []
+```
+
++ During appchain registration, the Token Icon of provided by appchain must be in SVG format.
```
Comment (lesterli) [inline src/guides/appchain-register.md]: Appchain team

File: src/guides/appchain-register.md [hunk 6]
```diff
@@ -1,8 +1,62 @@
...
+);
+```
+
++ The block time of the Appchain is consistent with the template, set to 6s.
+
+```rust
+pub const MILLISECS_PER_BLOCK: Moment = 6000;
+pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 4 * HOURS;
+```
++ In Appchain chain spec, ss58Format is 42, if you do not need to transfer the stablecoin assets (USDN) of the main chain (NEAR network) to Appchain, please set `assetIdByTokenId` with `[]`.
```
Comment (lesterli) [inline src/guides/appchain-register.md]: In Appchain chain spec JSON file, the setting of `ss58Format` should be 42. Also, if Appchain does not support the stablecoin assets (USDN in the NEAR network) cross-chain transfer from the Mainchain to Appchain, please set `assetIdByTokenId` with `[]`.

**Pull request merged**